lib/ramble/ramble/language/modifier_language.py

```diff
@@ -391,3 +391,35 @@ def _env_var_modification(mod):
             mod.env_var_modifications[mode][method].append(append_dict.copy())
 
     return _env_var_modification
+
+
+@modifier_directive(dicts=())
+def maintainers(*names: str):
```

Comment:
Similar comment to recent PR, I don't love how we duplicate so much stuff in the different "languages" but we can deal with that elsewhere

Reply:
Agreed (and I made a task for this already)